castervoice/lib/settings.py

```diff
@@ -449,11 +451,31 @@ def settings(key_path, default_value=None):
     return value
 
 
-def save_config():
+def save_config(paths = False):
     """
     Save the current in-memory settings to disk
     """
-    _save(SETTINGS, _SETTINGS_PATH)
+    guidance.offer()
+    if not paths:
+        # Use the paths on disk
+        result = {}
+        try:
+            with io.open(_SETTINGS_PATH, "rt", encoding="utf-8") as f:
+                result = tomlkit.loads(f.read()).value
+        except ValueError as e:
+            printer.out("\n\n {} while loading settings file: {} \n\n".format(repr(e), _SETTINGS_PATH))
+            printer.out(sys.exc_info())
+        except IOError as e:
+            printer.out("\n\n {} while loading settings file: {} \nAttempting to recover...\n\n".format(repr(e), _SETTINGS_PATH))
+        SETTINGS_tmp = copy.deepcopy(SETTINGS)
+        if "paths" in result:
+            SETTINGS_tmp['paths'] = result['paths']
+        _save(SETTINGS_tmp, _SETTINGS_PATH)
+    else:
+        _save(SETTINGS, _SETTINGS_PATH)
+
+     
+    
```

Comment:
Right now, paths only get written onto settings.toml if they are brand new defaults. 

I think that probably we don't have a use case for overwriting settings paths from within caster? So we could remove the if statement here and always run the first chunk of code?